### DISCUSSION TITLE
How to grant Unattended access to users without revealing password (shared address book with preset password)

### DISCUSSION THREAD

**Discussion by spacefrogfeds** [rustdesk/rustdesk-server-pro#405] [General]
Hi, We have staff members who need to access work PCs from home. We will be creating user accounts for all of them. And assigning them access to specific PCs. I assumed we could save the password into the address book and share it. But It appears that the address book doesnt hold passwords. The user will have to enter the password for each computer at least once. Is there an easier way to do this?

I just want the user to download rustdesk from our website, log in with their details, and access a set list of computers without needing a password.

**Comment by rustdesk** [comment]
@21pages

**Comment by 21pages** [comment]
Is there a key icon on the web console and client? 

And when connected with the ID input (A), the shared password will not be used, you need to click the device card (B) to connect, because the device may be in multiple shared address books. It can be improved when it only has one address book password.

![image](https://github.com/user-attachments/assets/47ef7b93-47c7-4aa5-9593-e69c1e93db6f)

![image](https://github.com/user-attachments/assets/733dce8d-2efc-48e3-9836-a35a660a4f52)

**Reply by spacefrogfeds** [reply]
Apologies for the late reply. I did find where I was going wrong. I found the option to add the password into an address book.

The issue I'm having now is that, If I want to share a PC across multiple users, or at a later date. I will have to re-enter the password into a new addressbook, unless I create an individual addressbook for each computer, and share multiple addressbooks to each user, which seems very clunky.

My understanding is that I should keep a record of each computers password. (or use 1 password for every computer, which is not a good idea)
